samba4 cross compile error...

```Traceback (most recent call last):
  File "./buildtools/bin/waf", line 75, in <module>
    import Scripting
  File "/home/captain/openwrt/build_dir/hostpkg/samba-4.9.11/third_party/waf/wafadmin/Scripting.py", line 146
    except Utils.WafError,e:
                         ^
SyntaxError: invalid syntax
make[3]: *** [Makefile:474: /home/captain/openwrt/build_dir/hostpkg/samba-4.9.11/.configured] Error 1
make[3]: Leaving directory '/home/captain/openwrt/feeds/packages/net/samba4'
time: package/feeds/packages/samba4/host-compile#0.26#0.10#0.34
make[2]: *** [package/Makefile:107: package/feeds/packages/samba4/host/compile] Error 2
make[2]: Leaving directory '/home/captain/openwrt'
make[1]: *** [package/Makefile:103: /home/captain/openwrt/staging_dir/target-x86_64_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/captain/openwrt'
make: *** [/home/captain/openwrt/include/toplevel.mk:218：world] 错误 2
```
subtarget:X86_64

--- a/net/samba4/Makefile
+++ b/net/samba4/Makefile
@@ -3,7 +3,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=samba
 PKG_VERSION:=4.9.11
-PKG_RELEASE:=2
+PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://ftp.heanet.ie/mirrors/ftp.samba.org/stable/ \
@@ -19,7 +19,7 @@ PKG_LICENSE:=GPL-3.0-only
 PKG_LICENSE_FILES:=COPYING
 
 # samba4=(asn1_compile) e2fsprogs=(compile_et) nfs-kernel-server=(rpcgen)
-HOST_BUILD_DEPENDS:=nfs-kernel-server/host e2fsprogs/host
+HOST_BUILD_DEPENDS:=python/host nfs-kernel-server/host e2fsprogs/host
 PKG_BUILD_DEPENDS:=samba4/host
 
 PKG_CONFIG_DEPENDS:= \

